.ci/install_cri_containerd.sh

```diff
@@ -41,4 +41,34 @@ if [ -n "$CI" ]; then
 	sudo mv /etc/cni/net.d/10-containerd-net.conflist  "$cni_test_dir"
 fi
 
+cat | sudo tee /etc/systemd/system/containerd.service  << EOT
```

Comment:
This looks wrong to me - shouldn't it be:

```
cat <<EOT | sudo tee /etc/systemd/system/containerd.service
```

Same comment for `0-crio.conf` below.